cranelift/codegen/src/isa/x64/pcc.rs

```diff
@@ -980,6 +980,8 @@ pub(crate) fn check(
         Inst::Unwind { .. } | Inst::DummyUse { .. } => Ok(()),
 
         Inst::StackSwitchBasic { .. } => Err(PccError::UnimplementedInst),
+
+        Inst::External { .. } => Ok(()), // TODO: unsure what to do about this!
```

Comment:
Totally fine to do `Err(PccError::UnimplementedInst)` for now -- I'll need to go and update this once we've moved instructions over that PCC cares about, but PCC isn't fuzzed at the moment anyway until I can make an update/fix pass...

Reply:
I tried that initially but I see test failures?

```
FAIL filetests/filetests/pcc/succeed/imul-fuzzbug.clif: compile

Caused by:
    Proof-carrying-code validation error: UnimplementedInst
```

To reproduce with the `Err(...)` version: `cd cranelift && cargo test --test filetests`.

Reply:
No `and` in that test but it's probably used in lowering some other instruction...

Reply:
Oh, I had forgotten we also had that test-suite still active -- OK, no worries, this is fine (and leave the TODO) and I'll get to it in my update pass!